Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+function ActionButton(props){
+    return(
+        <button className={props.btn.buttonStyle}>{props.btn.text}</button>
```

Comment:
This looks like a fine opportunity for [object destructuring](); for example:
```javascript
function ActionButton(props) {
  const { buttonStyle, text } = props.btn;
  return <button className={buttonStyle}>{text}</button>;
}
```
or, even shorter:
```javascript
function ActionButton({ btn }) {
  const { buttonStyle, text } = btn;
  return <button className={buttonStyle}>{text}</button>;
}
```